Updated join us to include new tabs and info

davidrkaufman opened the pull request:
I will celebrate if I didn't mess this up entirely

davidrkaufman committed:
Added Three New Press Stories to Press Section

chrisfrommann committed:
Fixed layout of press section

davidrkaufman committed:
Updated join us to include new tabs and info

I will celebrate if I didn't mess this up entirely

chrisfrommann committed:
Merge branch 'davidrkaufman-patch-1' of github.com:usds/website into davidrkaufman-patch-1

chrisfrommann committed:
Fixed some divs that weren't closed and formatting errors

chrisfrommann commented:
@davidrkaufman, made some fixes (there were some erroneously closed tags). Flag: there doesn't appear to be anything in the 'benefits' section?

davidrkaufman commented:
Fixed and added new pr

From: Chris Frommann [mailto:notifications@github.com]
Sent: Tuesday, October 25, 2016 2:30 PM
To: usds/website website@noreply.github.com
Cc: Kaufman, David R. EOP/OMB David_R_Kaufman@omb.eop.gov; Mention mention@noreply.github.com
Subject: Re: [usds/website] Updated join us to include new tabs and info (#206)

@davidrkaufmanhttps://github.com/davidrkaufman, made some fixes (there were some erroneously closed tags). Flag: there doesn't appear to be anything in the 'benefits' section?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHubhttps://github.com/usds/website/pull/206#issuecomment-256128670, or mute the threadhttps://github.com/notifications/unsubscribe-auth/AP8w6-tZJDYpEysAeUh1Ihj_pfbQXi-nks5q3krCgaJpZM4KgFW-.

davidrkaufman commented:
@chrisfrommann : https://github.com/usds/website/pull/208
